server/index.js

```diff
@@ -1,17 +1,24 @@
 // Dependencies
 const express = require('express')
+const bodyParser = require('body-parser');
+const { graphqlExpress, graphiqlExpress } = require('graphql-server-express');
+const schema = require('./api/schema');
+const cors = require('cors');
 
 // Dependencies - routes
-const rest = require('./apirest/rest')
+const rest = require('./apirest/rest');
 
 // App
 const app = express()
+app.use('*', cors());
 
 // Routes
-app.get('/', function (req, res) { res.status(400).send('invalid') })
+app.get('/', function (req, res) { res.status(400).send('invalid') });
```

Comment:
Use 404 status here instead. You don't need to send anything. You can leave this method empty: ```.send()```